source/gui/settingsDialogs.py

```diff
@@ -3129,6 +3144,9 @@ def onOpenScratchpadDir(self,evt):
 		path=config.getScratchpadDir(ensureExists=True)
 		os.startfile(path)
 
+	def onWasapiChange(self, evt):
+		self.soundVolFollowCheckBox.Enable(evt.IsChecked())

```

Comment:
should this also be set to false if wasapi is disabled?

```suggestion
		wasapiEnabled = evt.IsChecked()
		self.soundVolFollowCheckBox.Enable(wasapiEnabled)
		if not wasapiEnabled:
			self.soundVolFollowCheckBox.SetValue(False)
```

Reply:
I'm not convinced we should do this. This means that even if the user previously enabled it, we'll lose that setting when they disable and re-enable WASAPI. If they're just trying to test something without WASAPI for some reason, I don't see why they should have this other setting reverted to default when they re-enable WASAPI.

That said, if you feel really strongly about this, I'll relent and make the change. :)